netutil.py

```diff
@@ -45,16 +45,46 @@ def scanConfiguration():
     """
     conf = {}
     nics = []
+    slave_nics = []
 
     for nif in getNetifList():
         if nif not in diskutil.ibft_reserved_nics:
             nics.append(nif)
 
+    # identify any LACP interface previously configured (commandline)
+    SYSVIRTUALNET = "/sys/devices/virtual/net"
+    virtual_devs = os.listdir(SYSVIRTUALNET)
+    for iface in virtual_devs:
+        mode_file = os.path.join(SYSVIRTUALNET, iface, "bonding/mode")
+        if not os.path.exists(mode_file):
+            logger.log("scanConfiguration: no file %r" % mode_file)
+            continue            # not a bonding iface
+        with open(mode_file) as fd:
+            lines = fd.readlines()
+            assert len(lines) == 1
+            if lines[0].strip() != "802.3ad 4":
+                logger.log("scanConfiguration: wrong bonding mode %r" % lines[0])
```

Comment:
Would it be useful to specify in the log that the expected bonding mode is LACP?